app/services/postal_validator.rb

```diff
@@ -1,257 +1,252 @@
+      # CG: /\A([a-zA-Z\d\s]){3,}\z/,
+      # CI: /\A([a-zA-Z\d\s]){3,}\z/,
+      # CM: /\A([a-zA-Z\d]){3,7}\z/,
+      # CO: /\A([a-zA-Z\d\s]){3,}\z/,
+      # CR: /\A\d{5}\z/,
+      # CU: /\A\d{5}\z/,
+      # KM: /\A([a-zA-Z\d\s]){3,}\z/,
+      # DJ: /\A([a-zA-Z\d\s]){3,}\z/,
+      # DM: /\A([a-zA-Z\d\s]){3,}\z/,
+      # EG: /\A([a-zA-Z\d\s]){3,}\z/,
+      # GQ: /\A([a-zA-Z\d\s]){3,}\z/,
+      # EL: /\A\d{5}\z/,
+      # ER: /\A([a-zA-Z\d\s]){3,}\z/,
+      # FJ: /\A([a-zA-Z\d\s]){3,}\z/,
+      # FR: /\A\d{5}\z/,
+      # GA: /\A\d{2}\z/,
+      # GD: /\A([a-zA-Z\d\s]){3,}\z/,
+      # GH: /\A([a-zA-Z\d\s]){3,}\z/,
+      # GM: /\A([a-zA-Z\d\s]){3,}\z/,
+      # GN: /\A([a-zA-Z\d\s]){3,}\z/,
+      # GI: /\A(GX11 1AA)\z/i,
+      # GR: /\A\d{3}([ ]\d{2})\z/,
+      # GY: /\A([a-zA-Z\d\s]){3,}\z/,
+      # HN: /\A(([a-zA-Z]){2}|\d{2})\d{4}\z/,
+      # IO: /\A\d{5}\z/,
+      # IR: /\A\d{5}\z/,
+      # JP: /\A\d{7}\z/,
+      # JM: /\A(JM)[A-Z]{3}\d{2}\z/i,
+      # KI: /\A([a-zA-Z\d\s]){3,}\z/,
+      # KN: /\A([a-zA-Z\d\s]){3,8}\z/,
+      # KP: /\A([a-zA-Z\d\s]){3,8}\z/,
+      # LI: /\A\d{4}\z/,
+      # LV: /\A([a-zA-Z]|\d){3,8}\z/,
+      # LY: /\A\d{5}\z/,
+      # LT: /\A([a-zA-Z]){2}(-)\d{4}\z/,
+      # LC: /\A([a-zA-Z\d\s]){3,}\z/,
+      # MC: /\A\d{5}\z/,
+      # MD: /\A(([a-zA-Z]){2})(|\s)\d{4}\z/,
+      # ME: /\A([a-zA-Z\d\s]){3,}\z/,
+      # MH: /\A\d{5}\z/,
+      # MR: /\A([a-zA-Z\d\s]){3,}\z/,
+      # MM: /\A([a-zA-Z\d\s]){3,}\z/,
+      # MW: /\A([a-zA-Z\d\s]){3,}\z/,
+      # MZ: /\A\d{4}\z/,
+      # NA: /\A\d{5}\z/,
+      # NP: /\A\d{6}\z/,
+      # NR: /\A([a-zA-Z\d\s]){3,}\z/,
+      # PT: /\A\d{4}([\-]\d{3})?\z/,
+      # PS: /\A\d{3}\z/,
+      # PA: /\A\d{6}\z/,
+      # PE: /\A\d{5}\z/,
+      # QA: /\A([a-zA-Z\d\s]){3,}\z/,
+      # RW: /\A([a-zA-Z\d\s]){3,}\z/,
+      # SC: /\A([a-zA-Z\d\s]){3,}\z/,
+      # SE: /\A\d{3}[ ]\d{2}\z/,
+      # SK: /\A[089]\d{2}[ ]?\d{2}\z/,
+      # SL: /\A([a-zA-Z\d\s]){3,}\z/,
+      # SB: /\A([a-zA-Z\d\s]){3,}\z/,
+      # SR: /\A([a-zA-Z\d\s]){3,}\z/,
+      # SO: /\A([a-zA-Z\d\s]){3,}\z/,
+      # SV: /\A([a-zA-Z\d\s]){3,}\z/,
+      # SD: /\A\d{5}\z/,
+      # ST: /\A([a-zA-Z\d\s]){3,}\z/,
+      # SY: /\A([a-zA-Z\d\s]){3,}\z/,
+      # SS: /\A\d{5}\z/,
+      # TD: /\A\d{5}\z/,
+      # TG: /\A([a-zA-Z\d\s]){3,}\z/,
+      # TO: /\A([a-zA-Z\d\s]){3,}\z/,
+      # TZ: /\A([a-zA-Z\d\s]){3,}\z/,
+      # TT: /\A\d{6}\z/,
+      # TW: /\A\d{3}\z/,
+      # UK: /\A([A-PR-UWYZ0-9][A-HK-Y0-9][AEHMNPRTVXY0-9]?[ABEHMNPRVWXY0-9]? {1,2}[0-9][ABD-HJLN-UW-Z]{2}|GIR 0AA)\z/i,
+      # UG: /\A([a-zA-Z\d\s]){3,}\z/,
+      # UM: /\A([a-zA-Z\d\s]){3,}\z/,
+      # UY: /\A([a-zA-Z\d\s]){3,}\z/,
+      # VC: /\A(VC)\d{4}\z/i,
+      # VI: /\A\d{5}\z/,
+      # VG: /\A([a-zA-Z\d\s]){3,}\z/,
+      # VN: /\A\d{6}\z/,
+      # VU: /\A([a-zA-Z\d\s]){3,}\z/,
+      # WS: /\A([a-zA-Z\d\s]){3,}\z/,
+      # YE: /\A([a-zA-Z\d\s]){3,}\z/,
+      # ZW: /\A([a-zA-Z\d\s]){3,}\z/
```

Comment:
These changes all look good to me! only change I would advocate is to delete the unused Regexes instead of commenting them out. we have a version control system, so we can skip back to that point in time anyway if we ever want to uncomment them, and we don't leave a lot of crufty unused code around the repo.